packages/transport-webtransport/src/index.ts

```diff
@@ -175,15 +196,19 @@ class WebTransportTransport implements Transport {
           cleanUpWTSession('remote_close')
         })
 
-      if (!await this.authenticateWebTransport(wt, localPeer, remotePeer, certhashes)) {
+      if (!await raceSignal(this.authenticateWebTransport(wt, localPeer, remotePeer, certhashes), options.signal)) {
         throw new Error('Failed to authenticate webtransport')
       }
 
+      if (options?.signal?.aborted === true) {
+        throw new AbortError()
```

Comment:
maybe i'm misunderstanding things but looking at https://www.npmjs.com/package/race-signal, if the signal is aborted, `options?.signal?.aborted === true` will never be truthy. Should we check for aborted signal inside the `!await raceSignal` block, or wrap in try {} with a check there?

Reply:
Letting `raceSignal` throw if the signal aborts is enough, I've updated it.